stacks_queues/stack.py

```diff
@@ -21,18 +25,19 @@ def pop(self):
             The Stack is empty.
             returns None
         """
-        pass
+        return self.store.remove_first()
 
     def empty(self):
         """ Returns True if the Stack is empty
             And False otherwise
         """
-        pass
+        return self.store.empty()
 
     def __str__(self):
         """ Returns the Stack in String form like:
             [3, 4, 7]
             Starting with the top of the Stack and
             ending with the bottom of the Stack.
         """
-        pass
+        String = "[" + str(self.store.first()) + "]"
+        return String
```

Comment:
`String` is a keyword (name of the String class) in Python, so you want to avoid using it as your variable name. Additionally, you're only returning the first element of the Stack here - the specification asks for the entire stack to be printed instead. The Stack tests don't cover this method which is why they didn't catch this.

Try taking advantage of the `str` method in the `LinkedList` class instead.